apollo-router/src/services/layers/persisted_queries/snapshots/apollo_router__services__layers__persisted_queries__tests__pq_layer_freeform_graphql_with_safelist_log_unknown_true@logs.snap

```diff
@@ -2,6 +2,9 @@
 source: apollo-router/src/services/layers/persisted_queries/mod.rs
 expression: yaml
 ---
+- fields: {}
+  level: INFO
+  message: Loaded 2 persisted queries.
```

Comment:
@Velfi, is this change related to a PR?
Or it is just an artifact of running tests locally?